src/views/Dog/Dog.test.js

```diff
@@ -1,8 +1,13 @@
 import { render, screen } from '@testing-library/react';
+import { MemoryRouter, Route } from 'react-router-dom/cjs/react-router-dom.min';
```

Comment:
careful of the wonky import - this /cjs/react-router-dom.min breaks netlify builds, but looks like it doesn't affect CI on github.